README.md

```diff
@@ -18,16 +18,29 @@ supported environments.
 The reference examples should reflect the desired format.  The IDE configurations may not support or respect this, 
 which is okay while we're developing the format and auto-formatters.
 
-## Installation
+# Goals, Intent and Working Agreement
 
-### IntelliJ IDEA
+## Goals
+We want to present a unified approach to code styling across our entire suite of development efforts, automating away
+the need to respond to deviations during integration of code from a large group of developers, both within and outside
+of the Sonatype Organization.
+
+## Intent
+All of the development environments used by developers contributing to Sonatype code should be applying the same agreed
+upon formatting rules consistently, so that we don't spend needless time aligning these rules along with the more
+substantial (and important!) logical changes we make on a daily basis.
+
+## Working Agreement
+- Any change to existing rules should be discussed and agreed upon by the larger group before implementation.

```

Comment:
I think we need to figure out what this means. On one hand I don't believe we want individuals (or a clique) making changes to the codestyle with a handful of approvals and limited socialization, but at the same time I don't believe we want to wait on approvals from every developer.

Referring to changing the existing style or covering new areas as opposed to fixing issues with a given formatter.

Reply:
@mpiggott I can add a note to post to mailing list?